src/handler.js

```diff
@@ -33,25 +35,77 @@ const publicHandler = (request, response) => {
       return;
     }
     response.writeHead(200, {"content-type": contentTypeMapping[extension]});
+    console.log('sending file')
     response.end(file);
   });
 
 
 }
 
-const onLoad = (request, response) =>{
-  //get data from the database
-  dbConnection.query('SELECT * FROM tododb', (err, result) => {
-    if (err) {
-      console.log("Error")
-      console.log(err);
-    } else {
-        console.log(result);
-        response.writeHead(200, {"content-type":"application/json"})
-        response.end(JSON.stringify(result.rows));
-    }
+
+const signUpHandler = (request, response) => {
+
+  let body = '';
+  request.on('data', (data) => {
+    body += data;
   });
-};
+
+  request.on('end', () => {
+
+    const { username, password } = JSON.parse(body)
+    bcrypt.genSalt(10, function(err, salt) {
+    bcrypt.hash(password, salt, function(hashErr, hashed) {
+
+        if (hashErr) {
+        response.statusCode = 500;
+        response.end('Error registering')
+        return
+      }
+      var userArray = [username, hashed];
+
+      dbConnection.query(`INSERT INTO users (username, password) VALUES ($1, $2)`,userArray,
```

Comment:
might be nicer to pull the string out as well, and also use `const` instead of `var` for `userArray` 
```js
const userArray = [userArray]

const SQLstring=`INSERT INTO users (username, password) VALUES ($1, $2)`

dbConnection.query(SQLString, userArray, (err, result) => { 

```